### DISCUSSION TITLE
Welcome to Site Discussions!

### DISCUSSION THREAD

**Discussion by wouerner** [SouJunior/Site#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CarlosJunioor** [comment]
This is very interesting, specially for Open-Source community like us !

Enjoying it so far.

**Comment by EltonCampos07** [comment]
Conheci o projeto recentemente.

Adoro a ideia de projetos open source.
No momento, eu estou conhecendo a galera e participando de algumas reuniões sobre o andamento do projeto.
Na espera de uma oportunidade para ajudar o pessoal no projeto.

Espero conhecer e trocar experiência com pessoal aqui.

**Comment by Eletromaximus** [comment]
Parabéns pela iniciativa! Por enquanto apenas observando o projeto. Quem sabe um eu possa contribuir!
